src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -529,7 +534,7 @@ public OpenIdConnectAuthenticationHandler(HttpClient backchannel)
             if (subjectClaimType == null)
             {
                 Logger.LogError(string.Format(CultureInfo.InvariantCulture, Resources.OIDCH_0041_Subject_Claim_Not_Found, identity.ToString()));
-                return ticket;
+                return null;
```

Comment:
Consider faulting here as this is a protocol error. Returning 'null' will give the next middleware a crack at the message.